src/plugins/expression_error/public/components/error_component.tsx

```diff
@@ -49,7 +49,7 @@ function ErrorComponent({ onLoaded, parentNode, error }: ErrorComponentProps) {
           <EuiIcon
             className="canvasRenderError__icon"
             onClick={handlePopoverClick}
-            style={{
+            css={{
```

Comment:
it looks like the height/with here is dynamic
```suggestion
            style={{
```